recipes/conda-devenv/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "conda-devenv" %}
+{% set version = "0.9.0" %}
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_rev: {{ version }}
+  git_url: https://github.com/ESSS/conda-devenv.git
```

Comment:
Tag on the repo is fine. We just prefer to download a tarball instead of cloning is all.